ListLinked/Node.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace LinkedList
+{
+    public class Node<T>
```

Comment:
This class is supposed to be used only for private purposes of LinkedList class. So you can move it to the same file and make private.